Fix filter2

Kit486759 opened the pull request:
Fix genre selection on filter2 page.

miyabitanimchi commented on src/data/genresList.js:
```diff
@@ -5,15 +5,15 @@ const genresList = {
-    "progressive",
-    "psychedelic",
-    "alternative",
+    "indie",
+    "grunge",
+    "industrial",
+    "hip-hop",
+    "prog",
+    "psych",
+    "alt",
```
@Kit486759 Thank you Kit! The selecting buttons on Edgy is working now. But all the names on `src/data/genresList.js` should match with the values on `src/components/userAccount/GenreOptions.js` . So we should keep the names `indie`, `prog`, `psych`, and `alt` as `indie-rock`, `progressive`, `psychedelic`, and `alternative`. Could you change those 4 names with the original one? 🙏

Kit486759 replied:
Sure, I will change it back.

miyabitanimchi reviewed (commented):
Thank you Kit! But also check the comment below

miyabitanimchi commented:
@Kit486759 Hi kit, I changed the name drum-and-bass to drumandbass. just in case, please check the UI when you have time, thank you!

Kit486759 commented:
> @Kit486759 Hi kit, I changed the name drum-and-bass to drumandbass. just in case, please check the UI when you have time, thank you!

Thank you its working well. 
I have cleaned the console log too

BlackStone8960 reviewed (approved):
It works fine! Thank you for quick fixes :)
